### ISSUE TITLE
Problems whith plugins "pdf_view geo_view geojson_view officedocs_view federgob"

### ISSUE THREAD

**Issue by gpjoan** [ckan/ckanext-qa#29]
When I add a dataset, in order to get the qa stars I have to deactivate this plugins of the production.ini: "pdf_view geo_view geojson_view officedocs_view federgob". Then I execute the instruction: "paster --plugin=ckanext-qa qa update --config=/etc/ckan/default/production.ini", and  then activate the plugins again.

If I don't have this, appears an error and I can't see the datasets. What's the problem? Is there any solution?

Thanks.

**Comment by davidread** [comment]
What's the full error message? What do you mean "I can't see the datasets"?

**Comment by gpjoan** [comment]
I can't see the new dataset. For example, I tried to add a new dataset named "decimo". The previous datasets are ok, but when I try to enter in the new dataset I see this image:
![server error](https://cloud.githubusercontent.com/assets/16761362/12396459/e2052dc0-be07-11e5-8c41-347e786c9cd7.png)

and I see this error in celery:
![error](https://cloud.githubusercontent.com/assets/16761362/12396482/f3d225b2-be07-11e5-949a-5ea15f25db97.png)

**Comment by davidread** [comment]
Have you started celery with the virtualenv activated?

**Comment by gpjoan** [comment]
Yes, and the same celery window is working OK when I deactivated de plugins in production.ini

**Comment by davidread** [comment]
I can't think why that would be the case then.

**Comment by KrzysztofMadejski** [comment]
Celery has a problem of fully loading environment (see my comment https://github.com/ckan/ckan/issues/3491). There is a need to manually setup routes config and i18n. Maybe some workaronund for plugins is needed as well..
